packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -478,7 +523,7 @@ class SliverChildListDelegate extends SliverChildDelegate {
   /// The [children], [addAutomaticKeepAlives], [addRepaintBoundaries],
   /// [addSemanticIndexes], and [semanticIndexCallback] arguments must not be
   /// null.
-  const SliverChildListDelegate(
+  SliverChildListDelegate(
```

Comment:
this constructors' documentation should suggest using the other constructor in the case of a single constant list (where you don't need reordering support)

Reply:
same as above

Reply:
I missed read the comment, will add comment

Reply:
added